keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -18,6 +18,7 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/kubeedge/kubeedge/common/constants"
```

Comment:
kubeedge own package should be placed in the third part. For example: line 28 29

Reply:
I've modified him

Reply:
I've modified it

Reply:
also need fix the lint error, and please squash the commits to only one.